job_definitions/notify_suppliers_of_new_questions_answers.yml

```diff
@@ -0,0 +1,56 @@
+{% set environments = ['preview', 'production'] %}
+---
+{% for environment in environments %}
+- job:
+    name: "notify-suppliers-of-new-questions-answers-{{ environment }}"
+    display-name: "Notify suppliers of new questions and answers - {{ environment }}"
+    project-type: freestyle
+    description: "Send email notifications to supplier users about new questions and answers."
+    parameters:
+      - string:
+          name: SUPPLIER_IDS
+          description: "List of supplier IDs to be emailed e.g. 3,4,6"
+      - bool:
+          name: DRY_RUN
+          default: false
+          description: "List notifications that would be sent without sending the emails"
+    scm:
+      - git:
+          url: git@github.com:alphagov/digitalmarketplace-scripts.git
+          credentials-id: github_com_and_enterprise
+          branches:
+            - master
+          wipe-workspace: false
+{% if environment == 'production' %}
+    triggers:
+      - timed: "0 8 * * *"
```

Comment:
This is *exactly* the same time as the "notify_suppliers_of_dos_opportunities" job.

I'd be tempted to change both jobs to `H 8 * * *` (or `H 8 * * 1-5` for the opportunities one) so they're not both kicking off together.